src/briefcase/console.py

```diff
@@ -21,6 +24,39 @@ def enabled(self):
     def enabled(self, enabled):
         self._enabled = enabled
 
+    @staticmethod
+    def titlecase(s):
```

Comment:
Generally, I'm not a fan of staticmethod. Using classes as a namespace is a hangover of languages like Java that don't have "top level" functions. In this case, there's "Console" specific about titlecase - it's a utility method that will work on any string. It makes more sense as a top level function so it can be invoked without having any illusion that it's part of Console.

Reply:
Reverted the input handling refactor. Simply moved this back to `new.py`.